contents/projectManagement/AgileDevelopment.md

```diff
@@ -0,0 +1,133 @@
+- [Kanban](https://www.atlassian.com/agile/kanban)
+- [Scrumban](https://www.agilealliance.org/what-is-scrumban/)
+- [Lean Software Development](https://leankit.com/learn/lean/principles-of-lean-development/)
+- [Extreme Programming (XP)](http://www.agilenutshell.com/xp)
+- [Feature-Driven Development (FDD)](http://agilemodeling.com/essays/fdd.htm)
+- [Dynamic Systems Development Method (DSDM)](http://dsdmofagilemethodology.wikidot.com/)
+
+In addition to such agile frameworks, there are also many concrete practices that support agile development. These practices span over aspects like requirements, design, coding, testing and risk management, potentially easing your transition to agile development:
+
+- [User Stories](http://www.agilemodeling.com/artifacts/userStory.htm)
+- [User Story Mapping](https://www.thoughtworks.com/insights/blog/story-mapping-visual-way-building-product-backlog)
+- [Backlogs](https://www.agilealliance.org/glossary/backlog/)
+- [Velocity Tracking](http://www.softwaretestingstudio.com/agile-velocity-sprint-metrics/)
+- [Timeboxing](https://www.telerik.com/blogs/the-importance-of-timeboxing-and-iterations-for-agile-planning)
+- [Pair Programming](https://www.versionone.com/agile-101/agile-software-programming-best-practices/pair-programming/)
+- [Test Driven Development (TDD)](https://www.agiletestingframework.com/atf/testing/test-driven-development-tdd/)
+- [Continuous Integration](http://istqbexamcertification.com/what-is-continuous-integration-in-agile-methodology/)
+- [Reflection loops](https://dzone.com/articles/reflection-loops-agile)
+
+Of course, your project does not have to be utilising **all** the frameworks/concrete practices in order to be agile in its development. Instead, great care should be taken to analyse and select the appropriate framework/concrete practices that will work best for the project.
+
+## Caveats: What can possibly go wrong?
+
+Given that agile development can bring about so many benefits to our software projects, we must not forget that things can still go very wrong when using agile development. Such unfortunate situations are known as [agile anti-patterns](https://age-of-product.com/agile-management-anti-patterns/), or [agile smells](https://medium.com/agile-government-leadership/when-good-scrum-goes-bad-identifying-bad-agile-smells-690e7a16b501), and some common problems include:
+
+- **Unfamiliarity with agile development**
+
+If the team is unfamiliar with agile techniques, you will not be able to reap the full benefits of agile development since agile development depends on tight collaboration between team members. Therefore, it is crucial to ensure that the entire team has good foundations in agile techniques before the transition to agile development. This can be achieved by investing early in training sessions for the team, and bring everyone in the team up-to-date with what agile development entails for the project.
+
```

Comment:
Indent under the bullet point?
>* **Unfamiliarity with agile development**
>  If the team is unfamiliar with agile techniques, you will not be able to reap the full benefits of agile development since agile development depends on tight collaboration between team members. Therefore, it is crucial to ensure that the entire team has good foundations in agile techniques before the transition to agile development. This can be achieved by investing early in training sessions for the team, and bring everyone in the team up-to-date with what agile development entails for the project